apps/firelens-stability/templates/s3-fargate-v04-05-2023/fluent-bit.conf

```diff
@@ -0,0 +1,49 @@
+[SERVICE]
+    # See:
+    # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html
+    # https://github.com/aws-samples/amazon-ecs-firelens-under-the-hood/tree/master/generated-configs/fluent-bit
+    HTTP_Server  On
+    HTTP_Listen  0.0.0.0
+    HTTP_PORT    2020
+    Flush        1
+    Grace        30
+    Log_Level    info
+    storage.path /var/log/flb-storage/
+
+{{#if definitions.segfault}}
+# Artificially trigger a segfault if configured
+[INPUT]
+    Name      exec
+    command   kill -11 `pgrep fluent-bit`
+{{/if}}
+
+[INPUT]
+    Name          tcp
+    Tag           ApplicationLogs
+    Listen        0.0.0.0
+    Port          5170
+    Format        none
+    Chunk_Size    32
+    Buffer_Size   64
+    storage.type  filesystem
+
+    # Important! Every INPUT section's Mem_Buf_Limit must be re-computed when
+    # the container memory's limit or the number of of INPUT sections change. If you
+    # update this value, confirm that the container memory limit in the task
+    # definition is correct. Also note that an additional INPUT section is
+    # configured by default, so count the INPUTs in this file and add 1.
+    # Mem_Buf_Limit <= Memory_limit / 2 / (number of INPUT sections + 1)
+    # 64MB = 512MB / 2 / (4 INPUT sections)
+    Mem_Buf_Limit 64MB
+
+{{#times definitions.s3_outputPluginCount}}
+[OUTPUT]
+    Name s3
+    Match ApplicationLogs
+    bucket {{definitions.s3LogDestinationBucketName}}
+    s3_key_format /{{managed.executionName}}-{{managed.executionId}}/{{managed.caseNameUnique}}/${RUN_ID}/%Y-%m-%d/$UUID
```

Comment:
Create test with all weird things
- Tag of n
- $index
- UUID

Anything else

Reply:
Added a test case with the following